src/jblockmain/Main.java

```diff
@@ -16,24 +18,32 @@
     public static void main(String[] args)
     {
 
-        Measurements measurements = new Measurements("Spreadsheet.txt", true);
+        Measurements measurements = new Measurements("SpreadsheetBigger.txt", true);
 
         // Create patterns
         for (int i = 0; i < measurements.getNames().size(); i++)
         {
             measurements.setMapNumber(i);
 
+            new File("./output/BBSkirt/").mkdirs();
```

Comment:
Avoid explicit coding of directory names. Make the creation of the directory part of the DXF writer. Also move the `./output/` bit inside the DXF writer too so that you just have:    

```
TrouserPattern bb_trouser = new TrouserPattern(measurements);    
bb_trouser.writeToDXF();    
````

Reply:
Will sort this